sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/module-info.java

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+module com.azure.ai.formrecognizer {
+    requires transitive com.azure.core;
+
+    exports com.azure.ai.formrecognizer;
+    exports com.azure.ai.formrecognizer.models;
+
+    opens com.azure.ai.formrecognizer.implementation to com.fasterxml.jackson.databind;
+    opens com.azure.ai.formrecognizer.models to com.fasterxml.jackson.databind;
+    opens com.azure.ai.formrecognizer.implementation.models to com.fasterxml.jackson.databind, com.azure.core;
```

Comment:
Why does it open models package to com.azure.core?

Reply:
Get this error without it
>Caused by: java.lang.IllegalAccessException: class com.azure.core.http.rest.ResponseConstructorsCache (in module com.azure.core) cannot access class com.azure.ai.formrecognizer.implementation.models.AnalyzeReceiptAsyncResponse (in module com.azure.ai.formrecognizer) because module com.azure.ai.formrecognizer does not export com.azure.ai.formrecognizer.implementation.models to module com.azure.core
	at java.base/jdk.internal.reflect.Reflection.newIllegalAccessException(Reflection.java:361)